bin/udb3.php

```diff
@@ -57,7 +57,7 @@
 $consoleApp = $app['console'];
 $consoleApp->setCatchExceptions(false);
 
-// An udb3 system user is needed for conclude and geocode commands.
+// An udb3 system user is needed for geocode commands.
```

Comment:
Nit: The commands for updating the status of a single offer or multiple offers also needs this system user.